packages/storage/src/errors/types/validation.ts

```diff
@@ -70,4 +71,8 @@ export const validationErrorMap: AmplifyErrorMap<StorageValidationErrorCode> = {
 	[StorageValidationErrorCode.InvalidStoragePathInput]: {
 		message: 'Input `path` does not allow a leading slash (/).',
 	},
+	[StorageValidationErrorCode.InvalidStorageBucket]: {
+		message:
+			'Unable to lookup bucket from provided name in Amplify configuration.',
```

Comment:
Nit: Save a few bytes since this gets bundled universally

```suggestion
			'Bucket not found in configuration.',
```

Reply:
The ask from API BR was to keep it verbose since the whole string friendly name thing is not easily evident.